utils/clubs.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
+
+const request = require('request');
+
+const mongoose = require('mongoose');
+var db = mongoose.connect(process.env.MONGODB_URI, {useNewUrlParser: true, useUnifiedTopology: true});
+
+const Club = require('../model/club');
+const Member = require('../model/member');
+const ftw = require('./ftw');
+
+function callName (senderId, callback) {
+    request({
+        url: "https://graph.facebook.com/v4.0/" + senderId,
+        qs: {
+            access_token: process.env.PAGE_ACCESS_TOKEN,
+            fields: "name"
+        },
+        method: "GET"
+    }, function (err, response, body) {
+        if (err)
+            console.log("Error getting user's name: " +  err);
+        else {
+            var bodyObj = JSON.parse(body);
+            var name = bodyObj.name;
+            callback(senderId, name);
+        }
+    });
+}
+
+// only person who can modify club metadata is club owner e.g. owner's sender id
+function addClub(senderId) {
+    // we will authenticate this club through previous approval on google sheets
+    // confirmValidClub(senderId)
+
+    callName(senderId, function (id, clubName) {
+	Club.create({club_id: senderId, club_name: clubName}, function (err, res) { 
```

Comment:
`callName` gets the name for `senderId`, not the page's name (which I'm assuming is what you want). It's probably easiest to manually add clubs to the database, since it's a one-time thing for each club.